modules/manually-gathering-logs-with-ssh.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// *installing/installing-gather-logs.adoc
+
+[id="installation-manually-gathering-logs-with-SSH_{context}"]
+= Manually gathering logs with SSH access to your host(s)
+
+Manually gather logs in situations where `must-gather` or automated collection
+methods do not work.
+
+.Prerequisites
+
+* You must have SSH access to your host(s).
+
+.Procedure
+
+. Collect the `bootkube.service` service logs from the bootstrap host using the
+`journalctl` command by running:
++
+----
+$ journalctl -b -f -u bootkube.service
+----
+
+. Collect the bootstrap host's container logs using the Podman logs. This is shown
+as a loop to get all of the container logs from the host:
++
+----
+$ for pod in $(sudo podman ps -a -q); do sudo podman logs $pod; done
+----
+
+. Alternatively, collect the host's container logs using the `tail` command by
+running:
++
+----
+$ sudo tail -f /var/lib/containers/storage/overlay-containers/*/userdata/ctr.log
```

Comment:
```
[core@ip-10-0-53-178 ~]$ sudo tail -f /var/lib/containers/storage/overlay-containers/*/userdata/ctr.log
tail: cannot open '/var/lib/containers/storage/overlay-containers/*/userdata/ctr.log' for reading: No such file or directory
tail: no files remaining
```
I could get the expected logs after switching to `root` user
```
[core@ip-10-0-53-178 ~]$ sudo su
[root@ip-10-0-53-178 core]# 
[root@ip-10-0-53-178 core]# tail -f /var/lib/containers/storage/overlay-containers/*/userdata/ctr.log
==> /var/lib/containers/storage/overlay-containers/1deac4799d781572396bf81ce775ba0d537a4aaa42ecb2604be850d1840246a6/userdata/ctr.log <==
2019-11-26T08:49:42.553960402+00:00 stdout F wrote /assets/ingress-operator-manifests/cluster-ingress-00-custom-resource-definition.yaml
2019-11-26T08:49:42.554721135+00:00 stdout F wrote /assets/ingress-operator-manifests/cluster-ingress-00-namespace.yaml

==> /var/lib/containers/storage/overlay-containers/41972d7c1688d9e003b32342a0bab73d81bd26ae3f7be8c195482687fa95f853/userdata/ctr.log <==
2019-11-26T08:49:28.423415200+00:00 stdout F Writing asset: /assets/kube-apiserver-bootstrap/manifests/configmap-csr-controller-ca.yaml
2019-11-26T08:49:28.423546796+00:00 stdout F Writing asset: /assets/kube-apiserver-bootstrap/manifests/configmap-sa-token-signing-certs.yaml
```

Reply:
Thank you! I updated the command